gson/src/main/java/com/google/gson/JsonElement.java

```diff
@@ -143,6 +144,7 @@ public JsonPrimitive getAsJsonPrimitive() {
    * @throws IllegalStateException if this element is of another type.
    * @since 1.2
    */
+  @CanIgnoreReturnValue
   public JsonNull getAsJsonNull() {
```

Comment:
That's fair. I had to do the `var unused = ...` in a few places in Google's source code that were for example calling `getAsBoolean()` to check that it was indeed a Boolean and not something else. Here there's basically no _other_ reason to call it, since it can only ever return `JsonNull.INSTANCE`.